appinventor/components/src/com/google/appinventor/components/runtime/ListPickerActivity.java

```diff
@@ -10,12 +10,8 @@
 
 import android.content.pm.ActivityInfo;
 import android.util.Log;
-import android.view.KeyEvent;
-import android.view.View;
-import android.view.WindowManager;
-import android.widget.AdapterView;
-import android.widget.ArrayAdapter;
-import android.widget.EditText;
+import android.view.*;
```

Comment:
@kkashi01 Please use explicit imports. (I suspect IntelliJ or Eclipse did this for you).